pulpcore/cli/common/__init__.py

```diff
@@ -149,6 +150,13 @@ def _debug_callback(level: int, x: str) -> None:
 main.add_command(debug)
 
 
+@main.command("shell")
+@click.pass_context
+def pulp_shell(ctx: click.Context) -> None:
+    """Activate an interactive shell-mode"""
+    make_click_shell(ctx.parent, prompt="pulp-cli>", intro="Starting Pulp3 CLI...").cmdloop()
```

Comment:
```suggestion
    make_click_shell(ctx.parent, prompt="pulp>", intro="Starting Pulp3 interactive shell...").cmdloop()
```
IMHO at this point it stops being a cli. ;)